docs/sources/flow/getting-started/run-grafana-agent-flow.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Run Grafana Agent Flow
+weight: 100
+---
+
+# Run Grafana Agent Flow
+
+This topic describes how to run Grafana Agent Flow.
+
+## Before you begin
+
+* [Install Grafana Agent][]
+
+[Install Grafana Agent]: {{< relref "../../set-up" >}}
+
+## Steps
+
+To run Grafana Agent Flow, follow these steps:
+
+1. Set the `AGENT_MODE` environment variable on your system to `flow`.
+
+2. Create a configuration file to Grafana Agent Flow to use, saving it to your
+   system. For example:
+
+   ```river
+   logging {
+     level  = "info"
+     format = "logfmt"
+   }
+   ```
+
+3. Run the following command in your terminal, replacing `FILE_PATH` with the
+   path to the configuration file you created:
+
+   ```bash
+   grafana-agent run FILE_PATH
```

Comment:
I noticed some old documentation still refers to `agent` instead of `grafana-agent`. I'll fix that in a follow up PR once this is merged.